xml/bp_troubleshooting_osds.xml

```diff
@@ -310,6 +321,7 @@ osd.0 is down since epoch 23, last address 192.168.106.220:6800/11080
   <sect2 xml:id="troubleshooting-drive-config">
     <title>Drive Configuration</title>
     <para>
+<!--       Nowadays the exception is fast SSDs and NVMEs -->
```

Comment:
Unsure what needs to be changed here? @jschmid1

Reply:
Imo, this is not troubleshooting but general guidance and I'm pretty sure we have paragraphs about that.

Reply:
Whole section removed :D